components/Common/DarkModeToggle/index.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import DarkModeToggle from '.';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof DarkModeToggle> = {
+  title: 'DarkModeToggle',
+  component: DarkModeToggle,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof DarkModeToggle>;
+
+export const Toggle: Story = {
+  render: () => <DarkModeToggle />,
```

Comment:
Do you need to provide this render method? You're already providing the `component`

I guess this can be changed to `export const Default: Story = {}`